src/main/resources/db/migration/V13__iat_ddl.sql

```diff
@@ -0,0 +1,12 @@
+--
+-- https://jira.fairwaytech.com/browse/IAT-3209
+--
+
+-- item history
+ALTER TABLE item_history
+    ADD COLUMN commit_by_fullname TEXT CHECK (char_length(commit_by_fullname) <= 255);
```

Comment:
Interesting length check syntax in psql

Reply:
From what I read, varchar and text are pretty much identical.  The advantage of text with a length check is it is easier to resize.  I'm not super clear on that but it was easier enough to go this route in case it ever came up.